### ISSUE TITLE
Typescript Syntax Errors

### ISSUE THREAD

**Issue by DanniB** [valor-software/ngx-bootstrap#998]
Hi there,

I'm facing issues concerning Typescript errors since I upgraded to Angular 2.0.0. At the same time I upgraded to Typescript 2.0.2. Does this relate to my gulp settings or is it a general problem? 

```
[09:06:59] Starting 'copy:libs'...
.../node_modules/ng2-bootstrap/components/buttons/button-checkbox.directive.d.ts(14,
22): error TS1005: '=' expected.
.../node_modules/ng2-bootstrap/components/buttons/button-checkbox.directive.d.ts(15,
22): error TS1005: '=' expected.
.../node_modules/ng2-bootstrap/components/buttons/button-radio.directive.d.ts(10,14)
: error TS1005: '=' expected.
.../node_modules/ng2-bootstrap/components/buttons/button-radio.directive.d.ts(10,22)
: error TS1005: ';' expected.
.../node_modules/ng2-bootstrap/components/dropdown/dropdown-toggle.directive.d.ts(11
,14): error TS1005: '=' expected.
.../node_modules/ng2-bootstrap/components/dropdown/dropdown-toggle.directive.d.ts(11
,20): error TS1005: ';' expected.
.../node_modules/ng2-bootstrap/components/modal/modal.component.d.ts(14,14): error T
S1005: '=' expected.
.../node_modules/ng2-bootstrap/components/modal/modal.component.d.ts(14,21): error T
S1005: ';' expected.
.../node_modules/ng2-bootstrap/components/modal/modal.component.d.ts(22,22): error T
S1005: '=' expected.
.../node_modules/ng2-bootstrap/components/position.d.ts(29,22): error TS1005: '=' ex
pected.
.../node_modules/ng2-bootstrap/components/position.d.ts(30,22): error TS1005: '=' ex
pected.
.../node_modules/ng2-bootstrap/components/tooltip/tooltip-container.component.d.ts(2
1,14): error TS1005: '=' expected.
.../node_modules/ng2-bootstrap/components/tooltip/tooltip-container.component.d.ts(2
1,24): error TS1005: ';' expected.
.../node_modules/ng2-bootstrap/components/typeahead/typeahead-container.component.d.
ts(18,14): error TS1005: '=' expected.
.../node_modules/ng2-bootstrap/components/typeahead/typeahead-container.component.d.
ts(18,26): error TS1005: ';' expected.
.../node_modules/ng2-bootstrap/components/typeahead/typeahead-container.component.d.
ts(18,44): error TS1005: '(' expected.
.../node_modules/ng2-bootstrap/components/typeahead/typeahead.directive.d.ts(46,14):
 error TS1005: '=' expected.
.../node_modules/ng2-bootstrap/components/typeahead/typeahead.directive.d.ts(46,21):
 error TS1005: ';' expected.
.../node_modules/ng2-bootstrap/components/typeahead/typeahead.directive.d.ts(46,33):
 error TS1005: '(' expected.
[09:07:09] TypeScript: 19 syntax errors

```

Kind regards,
Danni

**Comment by wilfredmifsud** [comment]
Happened to me as well

**Comment by adrianfaciu** [comment]
You need to instruct gulp to use a different ts version. By default it uses a bundled one, which is 1.8.x.
Have a look at the closed issues (#982), you'll find several with this issue.
